app/src/main/java/io/gnosis/safe/ui/settings/app/passcode/ChangeRepeatPasscodeFragment.kt

```diff
@@ -81,6 +82,16 @@ class ChangeRepeatPasscodeFragment : BaseViewBindingFragment<FragmentPasscodeBin
             input.doOnTextChanged(onSixDigitsHandler(digits, requireContext()) { digitsAsString ->
                 if (digitsAsString == passcodeArg) {
                     viewModel.disableAndSetNewPasscode(newPasscode = passcodeArg, oldPasscode = oldPasscode)
+                    val cm = CryptographyManager()
```

Comment:
cryptography manager operations should not be in fragment. I suggest to add this code to HeimdallEncryptionManager and use it in a view model of the fragment